src/config.rs

```diff
@@ -29,12 +29,14 @@ pub struct Config {
 
     #[get = "pub"]
     #[clap(
-        default_value("/var/run/cri/cri.sock"),
-        env("CRI_SOCK_PATH"),
-        long("sock-path")
+        default_value("/var/run/cri"),
+        env("CRI_RUN_PATH"),
+        long("run-path"),
```

Comment:
I feel like we should keep the storage and socket/listen path separate.

Reply:
Sure, added the storage path as separate config value.